### ISSUE TITLE
[a11y] Improve light theme colors

### ISSUE THREAD

**Issue by noelmace** [blindnet-io/blindnet.dev#17]
Links [are not standing out enough](https://github.com/blindnet-io/blindnet.dev/pull/16#issuecomment-1160571691)

## Context

Before #16:

![image](https://user-images.githubusercontent.com/7578400/174637925-8b884cd9-3898-473d-b14f-6077b467b72d.png)

![image](https://user-images.githubusercontent.com/7578400/174638261-b5c65902-a38f-4246-a907-a486c3cd9758.png)

After #16:

![image](https://user-images.githubusercontent.com/7578400/174638110-1c3a78c1-5a34-451f-a1cf-2dad0468928a.png)

![image](https://user-images.githubusercontent.com/7578400/174638348-facd8a45-6dd8-4d26-b220-e5b397d4cb83.png)

**Comment by m4rk055** [comment]
I'd propose a few improvements to the dark theme.

Soothe the color of the **Caution** admonition as (at least for me) it's difficult to read.

![image](https://user-images.githubusercontent.com/71560916/174757445-071fc888-dccf-4c21-b934-a1ba4044c2e3.png)

Purple/pink syntax colors (**dracula** theme) does not look so nice on the current background. I prefer the **vsDark** theme

![image](https://user-images.githubusercontent.com/71560916/174761182-3d000c32-7d99-45e2-86dc-007f94d09809.png)

Content tabs color stands out a lot.
![image](https://user-images.githubusercontent.com/71560916/174761457-2ea99af8-1625-4c7f-a379-35861a9289ab.png)
I prefer the original
![image](https://user-images.githubusercontent.com/71560916/174761475-9523c548-ce81-4ad7-b33c-8f6496b97385.png)

**Comment by noelmace** [comment]
Thx @m4rk055 for this feedback.

Most points aren't related to this issue though:

1. OK a11y wise (see https://color.a11y.com/ContrastPair/?bgcolor=4d3800&fgcolor=ffffff), so I don't want to spend time discussing it for now. Please open a new issue if you want to discuss the dark theme colours aesthetic in more detail (as I still believe this could be useful).
2. I really like that :+1: Please send a PR, or create a new issue if you don't have time for this now.
3. This new theme colour is a brand colour and will be kept as is to match the blindnet.io brand colour, as explained in the original PR. The original was Docusaurus' brand colour, which could create confusion. If you still believe we should discuss the choice of the brand color, let me know, and I'll create an associated DR draft.
